src/client/Microsoft.Identity.Client/Cache/Prototype/IdentityCacheWrapper.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Identity.ServiceEssentials;
+
+namespace Microsoft.Identity.Client.Cache.Prototype
+{
+    internal class IdentityCacheWrapper
+    {
+        internal static IIdentityCache s_iIdentityCache { get; set; }
```

Comment:
This should probably be updated. The lifecycle of user-provided cache should be managed externally to MSAL. Whether the default cache is static or not would be set via `CacheOptions.UseSharedCache`.